### PR TITLE
London Class 7 - Kara Howard - JavaScript Core 1 - Week 2

### PR DESCRIPTION
**Pull request by kmh256**
# Your Details

- Your Name: Kara Howard
- Your City: London
- Your Slack Name: Kara Howard

# Homework Details

- Module: JavaScript Core 1
- Week: Week 2

### CODE REVIEW

File: exercises/C-comparison-operators/exercise.js [hunk 1]
```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
```
Comment (kmh256) [inline exercises/C-comparison-operators/exercise.js:10]: updated to: var enoughSpaceInRoom = roomMaxCapacity >= (studentCount + mentorCount);

File: exercises/C-comparison-operators/exercise.js [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity > moreStudentsThanMentors; // finish this statement
```
Comment (kmh256) [inline exercises/C-comparison-operators/exercise.js]: updated to: var enoughSpaceInRoom = roomMaxCapacity >= (studentCount + mentorCount);

File: exercises/D-predicates/exercise.js [hunk 3]
```diff
@@ -6,10 +6,14 @@
...
-function isNegative(number) {}
+function isNegative(number) {
+  return number < 0;
+}
 
 // Finish the predicate function to test if the passed number is between 0 and 10
-function isBetweenZeroAnd10(number) {}
+function isBetweenZeroAnd10(number) {
+  return number > 0 && number < 10;
+}
```
Comment (AhmadJanah) [inline exercises/D-predicates/exercise.js:16]: Hi Kara,

You did great work, congratulation,

I think "between" should include the numbers of 0 and 10 as well, so your code will be:

return number >= 0 && number <= 10;

What do you think?

File: exercises/G-conditionals-2/exercise-3.js [hunk 4]
```diff
@@ -8,7 +8,16 @@
...
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return `A`;
+  } else if (mark < 80 && mark > 60) {
+    return `B`;
+  } else if (mark <= 60 && mark >= 50) {
+    return `C`;
+  }
+  return `F`;
+}
```
Comment (AhmadJanah) [inline exercises/G-conditionals-2/exercise-3.js:20]: Your work here is definitely correct, but you can make it shorter by removing some code, for example you can remove (mark < 80 && ) from second if, as the compiler will come here only when mark < 80. your code can be:
if(mark >= 80){
return "A";
} else if (mark > 60) {
return "B";
} else if (mark >50) {

**Review by AhmadJanah — COMMENTED** [review]
Thank you Kara for this great work, I hope you will accept my comments leniency.
Your code is amazing, but I added some extra improvements.

Best Wishes,

File: exercises/D-predicates/exercise.js (hunk 3 above)
Comment (kmh256) [reply exercises/D-predicates/exercise.js:16]: Thank you Ahmad! I have corrected my code :)

File: exercises/G-conditionals-2/exercise-3.js (hunk 4 above)
Comment (kmh256) [reply exercises/G-conditionals-2/exercise-3.js:20]: Thank you for your feedback Ahmad! It has been very helpful :)

**Pull request closed without merging**